README.md

```diff
@@ -16,13 +16,13 @@ pipenv install --dev
 Extract translatable text from an eQ survey with
 
 ```
-pipenv run ./scripts/run_translation_extract.sh <json_file> [output_directory]
+pipenv run ./scripts/run_translation_extract.sh <json_file> [output_directory] pot/csv
```

Comment:
Can't run as it is failing on:
https://github.com/ONSdigital/eq-translations/blob/29f8b644c5232189fd7ae9c1ffd7d3d6342fb58c/scripts/run_translation_extract.sh#L39

`extract_translation_data.py` doesn't exist.

Tried to point it to `template_extractor.py` but failing on imports:
```
Traceback (most recent call last):
  File "./app/template_extractor.py", line 15, in <module>
    from app.utils import get_output_file_path, deserialise_json
ModuleNotFoundError: No module named 'app'
```